physionet-django/physionet/settings/production.py

```diff
@@ -18,6 +18,7 @@
         'PASSWORD': config('DB_PASSWORD'),
         'HOST': config('DB_HOST'),
         'PORT': config('DB_PORT', default=''),
+        'DISABLE_SERVER_SIDE_CURSORS': config('DISABLE_SERVER_SIDE_CURSORS', default=False),
```

Comment:
I would add `cast=bool` as for example in here: https://github.com/MIT-LCP/physionet-build/blob/dev/physionet-django/physionet/settings/base.py#L47